### ISSUE TITLE
Add folders, favorites, types separation (login, ID, card, etc)

### ISSUE THREAD

**Issue by mxmvncnt** [Bitsteward/bitsteward#5]
Separate everything into submenus. The structure would be like so:

All vault items/
├─ item 1
├─ item 2
├─ item 3 (also a fav.)
├─ item 4
Favorites/
├─ Item 3
Logins/
├─ Item 2
Folder 1/
├─ Item 3
├─ Item 4
...

The first view shows a GtkSidebar with the top folders like All vault items, then when it is pressed, the sidebar changes to show the content inside that "folder". Then the user can choose which vault item to show by selecting it. 

Basically a double leaflet that is always folded, kind of like the way the GNOME Settings app handles submenus in the side-bar
